chef-utils/lib/chef-utils/dsl/virtualization.rb

```diff
@@ -140,6 +140,26 @@ def vmware_host?(node = __getnode)
         node.dig("virtualization", "system") == "vmware" && node.dig("virtualization", "role") == "host"
       end
 
+      # Determine if the current node is virtualized on VMware Desktop (Fusion/Player/Workstation).
+      #
+      # @param [Chef::Node] node
+      #
```

Comment:
Can you add the yard `@since` to these for 17.9. We generate the vscode plugin off this file and use those to document when a helper became available.